packages/playground/windows/playground/pch.h

```diff
@@ -17,13 +17,8 @@
 #include <winrt/Windows.Foundation.Collections.h>
 #include <winrt/Windows.Foundation.Metadata.h>
 #include <winrt/Windows.Foundation.h>
```

Comment:
Should we remove them too?

Reply:
once we make winui3 the default we can start trimming down cppwinrtincludes and reducing the PCH, so i think leaving these here is fine as they'll end up here anyway